Source/svg/SVGParser.swift

```diff
@@ -1730,3 +1785,18 @@ fileprivate class UserSpaceNode {
         self.userSpace = userSpace
     }
 }
+
+fileprivate class UserSpacePattern: Fill {
```

Comment:
Can we remove this class at all? I think it's dangerous to create `Fill` implementation which actually can't be rendered, but necessary for parsing purpose. If you need to store it in `defFills`, just create a separate class like it was done for `defMasks` and `defClip`.